docs/TSO.md

```diff
@@ -1,62 +1,62 @@
 [-UP-](./TXN.md)
 
-# K2 Timestamp and TSO 
+# K2 Timestamp and TSO
 ## 1.  Design goals
```

Comment:
Small change in renaming the APIs to conform to the rest of the repo: private methods start with `_` and all methods are in `camelCase` instead of `PascalCase`